schema/workflow.yaml

```
@@ -1,5 +1,5 @@
 $id: https://serverlessworkflow.io/schemas/1.0.0-alpha1/workflow.json
```

```suggestion
$id: https://serverlessworkflow.io/schemas/1.0.0-alpha1/workflow.yaml
```

We might need to change it in the `.ci` validation too.